src/runtime/internal/keyed_each.ts

```diff
@@ -112,7 +112,7 @@ export function validate_each_keys(ctx, list, get_context, get_key) {
 	for (let i = 0; i < list.length; i++) {
 		const key = get_key(get_context(ctx, list, i));
 		if (keys.has(key)) {
-			throw new Error('Cannot have duplicate keys in a keyed each');
+			throw new Error(`Cannot have duplicate keys in a keyed each: ${key} is a duplicate`);
```

Comment:
wrap around with quotes to have a clearer idea of when the key starts and ends, especially when having string value as key.

Also, i would suggest add `JSON.stringify()` to differentiate between number and string value key

```suggestion
			throw new Error(`Cannot have duplicate keys in a keyed each: \`${JSON.stringify(key)}\` is a duplicate`);
```